File: matlab_tests/simple_test.py
import numpy as np

def simple_test(x):
    # Basic arithmetic
    a = (x ** 2)
    b = np.sqrt(a)
    # MATLAB built-in conversion test
    c = np.sin(x)
    # Loop test
    total = 0
    for i in range(1, 5+1):
        total = (total + i)
    # Final output
    y = ((b + c) + total)
    
    return y

File: matlab_tests/simple_test.m
function y = simple_test(x)

% Basic arithmetic
a = x^2;
b = sqrt(a);

% MATLAB built-in conversion test
c = sin(x);

% Loop test
total = 0;

for i = 1:5
    total = total + i;
end

% Final output
y = b + c + total;

end